src/main/java/org/isf/login/rest/LoginController.java

```diff
@@ -64,7 +64,7 @@ public class LoginController {
 	private TokenProvider tokenProvider;
 	
 	@Autowired
-        private CustomAuthenticationManager authenticationManager;
+    private CustomAuthenticationManager authenticationManager;
```

Comment:
Formatting looks incorrect

Reply:
i don't have this formatting looks on my side